examples/filesystem/main.c

```diff
@@ -26,6 +26,21 @@
 #include "shell.h"
 #include "board.h" /* MTD_0 is defined in board.h */
 
+#if !defined(MTD_0) && MODULE_MTD_SDCARD
+#include "mtd_sdcard.h"
+#include "sdcard_spi.h"
+#include "sdcard_spi_params.h"
+
+#define SDCARD_SPI_NUM ARRAY_SIZE(sdcard_spi_params)
+extern sdcard_spi_t sdcard_spi_devs[SDCARD_SPI_NUM];
+mtd_sdcard_t mtd_sdcard_devs[SDCARD_SPI_NUM];
+
+/* always default to first sdcard*/
+static mtd_dev_t *mtd0 = (mtd_dev_t*)&mtd_sdcard_devs[0];
+
+#define MTD_0 mtd0
+#endif
```

Comment:
Could you add some comments here, similar to the other blocks to explain why every bit is required for SD card operation. It makes the example a bit more useful.

Reply:
I also brought it more in line with other MTD examples.